client/py/tests/test_auto.py

```diff
@@ -0,0 +1,70 @@
+
+import pytest
+import numpy as np
+import pandas as pd
+
+import synnax as sy
+from synnax.simulation.auto import Simulator
```

Comment:
You can group all these imports together i.e. `from synnax.simulation.auto import One, Two`